Resources/views/backend/swag_backend_order/controller/main.js

```diff
@@ -617,6 +620,12 @@ Ext.define('Shopware.apps.SwagBackendOrder.controller.Main', {
 
                 me.validationMail = pluginConfigObj.data.validationMail;
                 me.desktopTypes = pluginConfigObj.data.desktopTypes;
+                me.sendMail = pluginConfigObj.data.sendMail;
+                if (me.sendMail == 1) {
+                    me.orderModel.set('sendMail', 1);
+                    //Set checked class for table
+                    document.getElementById('sendMailChk').classList.add('x-form-cb-checked');
```

Comment:
Hey guys, 
thanks for your ideas. At first I wanted to use the comment of @mitelg 
But the value is in relation to the plugin config. My idea was to use following: 

```
if(me.orderModel.get('sendMail'))
   me.sendMailCheckbox.setValue(true);
```

But this doesn´t work because the orderModel is always to false. I think this is because the plugin config in the main.js is loaded asynchronously. If I do this with a setTimeout it actually works :) Maybe someone of you have an idea how to get the plugin configuration within this context?